beangulp/importers/csv.py

```diff
@@ -274,6 +277,9 @@ def get(row, ftype):
                 last4 = get(row, Col.LAST4)
 
                 balance = get(row, Col.BALANCE)
+                currency = self.currency \
+                    if Col.CURRENCY not in iconfig or not get(row, Col.CURRENCY) \
+                    else get(row, Col.CURRENCY)
```

Comment:
It seems that this would be easier to unwrap into a regular `if ... else` block.